src/types/form/AddDebitCardForm.ts

```diff
@@ -0,0 +1,30 @@
+import type Form from './Form';
+
+const INPUT_IDS = {
+    SETUP_COMPLETE: 'setupComplete',
+    NAME_ON_CARD: 'nameOnCard',
+    CARD_NUMBER: 'cardNumber',
+    EXPIRATION_DATE: 'expirationDate',
+    SECURITY_CODE: 'securityCode',
+    ADDRESS_STREET: 'addressStreet',
+    ADDRESS_ZIP_CODE: 'addressZipCode',
+    ADDRESS_STATE: 'addressState',
+    ACCEPT_TERMS: 'acceptTerms',
+} as const;
+
+type AddDebitCardForm = Form<{
```

Comment:
NAB / can be done in a follow-up, but there's nothing here to guarantee that the keys of the form type for a given form match the `INPUT_IDS` for that form. I can add keys and remove them and there's no errors.